terratest/examples/broken-gslb.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: k8gb.absa.oss/v1beta1
+kind: Gslb
+metadata:
+  name: broke-test-gslb
+spec:
+  ingress:
+    rules:
+      - host: terratest-failover.cloud.example.com
+        https:
```

Comment:
should we also test for missing `http` in addition to misspelled `http(s)`?

Reply:
sure, I can add it

Reply:
@somaritane please check